inspirehep/ext/redis/__init__.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
```

Comment:
Ah you mean the whole thing, yeah, right that can be removed. I thought you meant the encoding :dancer: